app/features/sidebar/pools-section/pool-item.tsx

```diff
@@ -1,10 +1,9 @@
 import getPoolContextMenu from "app/pools/flows/get-pool-context-menu"
-import {lakeSearchPath} from "app/router/utils/paths"
+import {poolSearchPath} from "app/router/utils/paths"
```

Comment:
shoot, I changed these paths in my query-flow work too, though we are using the same name for this one 👌